src/domains/ecommerce-app/api/ecommerce-io/v2/_openapi.json.tpl

```diff
@@ -773,6 +781,74 @@
           }
         }
       }
+    },
+    "/users/{userId}/stats": {
```

Comment:
the API group (ecommerce for IO) is dedicated to pagoPA payments for IO and does not have all the apis exposed for IO of the pagoPA platform. Maybe this api is too generic? it is unlikely that we will add statistics related to BIZ rather than GEC to this group. This API group is dedicated to payment flows, so I would reinforce this concept in the API signature.

Reply:
It's a good point. I put this api here because it is really part of the payment flux, like others api to retrieve other payment methods. I have some ideas about its name, but I will get a survey to the team to take a definitive choice